### ISSUE TITLE
"Rescan All" button hides bottom part of list

### ISSUE THREAD

**Issue by McTwist** [Catfriend1/syncthing-android#689]
### Description of the issue
The Rescan All button at the bottom hides part of devices (and presumable folders) list if it has too many items.
### Reproducer
Have enough devices or folders to fill up the list enough.
### Version Information
    App Version: 1.8.0.2
    Syncthing Version: v1.8.0
    Android Version: Android 10 / OxygenOS 10.0.0
    Device manufacturer: OnePlus
    Device model: A5000

### Device platform info
Irrelevant.

### Android Log
Irrelevant.

**Comment by Catfriend1** [comment]
Yes, it's early for that bar and I'm a bad UI designer :(... There's definitely room for improving it in the future. E.g. offering different buttons while on folder / device tab, showing it on long press etc. Putting the open folder to it. The start wasn't clear, because every pro comes with cons, e.g. I thought a long time ago the bar breaks tv compatibility. So love to hear your feedback (and from others). Should we keep and improve? Ditch it and refer to web UI? Give us time ;)

**Comment by McTwist** [comment]
I would recommend to put it to the right as a circle. The design of the icon works good with that as well, and even if it would block parts of the right part, it doesn't matter as that is mostly repetitive information/icons.

**Comment by vohe** [comment]
I would recommend to put it into the menu. I never need this, because syncthing does this automatically.

**Comment by Catfriend1** [comment]
Basically a good idea. But the menu is difficult as well if you consider small screens. Or we let some users be lucky and only show the bar when expert options are enabled?

**Comment by vohe** [comment]
At the moment it makes more problems that it would fix! Please can you move it to the top (where the new folder button lives?!)

**Comment by McTwist** [comment]
I don't have a small screen, so I don't know how that works, but if put in the left menu and it is able to scroll, then there wouldn't be any problem, right? Smaller screens just have to scroll down to reach that part. This feature is helpful, but only if it is accessible enough. If moved too deep into a menu, it would be faster to restart the app manually by terminating it.

**Comment by vohe** [comment]
I think this function is used so seldom that this big button is unnecessary. Rescan is defined several times in seconds. And normally you only need the scan when setting up a folder.
I can only imagine the need for scenarios in which users use the app as static synchronization. From my point of view, that would mean that they did not understand the system of continuous synchronization

**Comment by McTwist** [comment]
I might be wrong here, but I assumed that there was two ways to synchronize the folder:

- Listening on the file system if any changes, and handle accordingly.
- Check once within a time limit (Normally one hour) and update where needed.

> Rescan is defined several times in seconds.
Could you explain what you mean with this, due to the previous assumption of functionality?

**Comment by Catfriend1** [comment]
Hmm .. I had a use case myself some days ago where Syncthing only picked up all changes correctly when restarting it (side menu) or hitting rescan. So it is useful as a service action. We could try move it to the top as well, let's see if everybody's phone screen is still big enough for that 👍

**Comment by McTwist** [comment]
I've noticed that devices disconnects occasionally, and a client restart has to be taken in action before it can discover the visible devices. This may have been fixed as I don't see this as often as I did before. May be that it is shown more often when the phone is roaming between mobile and wifi.

The issue that folders does not sync have happened before, and this button is nice to have if one discovers that nothing have been transferred.
